cmd/terramate/cli/cli.go

```diff
@@ -72,9 +72,10 @@ type cliSpec struct {
 	DisableCheckGitUncommitted bool `optional:"true" default:"false" help:"disable git check for uncommitted files."`
 
 	Run struct {
-		ContinueOnError bool     `default:"false" help:"continue executing in other stacks in case of error."`
-		DryRun          bool     `default:"false" help:"plan the execution but do not execute it"`
-		Command         []string `arg:"" name:"cmd" passthrough:"" help:"command to execute."`
+		DisableCheckGenCode bool     `optional:"true" default:"false" help:"disable outdated generated code check."`
```

Comment:
The check is only performed on terramate run, so it made sense to add the flag only for run.